testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -365,16 +365,6 @@ QUnit.module('dropDownOptions', () => {
         assert.strictEqual(instance._popup.option('showTitle'), true, 'Option has been redefined');
     });
 
-    QUnit.test('dropDownOptions should be prior than built-in public options', function(assert) {
```

Comment:
Why do we remove this test?

Reply:
Cause we deprecate showPopupTitle

Reply:
I think we should remove it with the implementation through 2 major releases.

Reply:
After deprecating showPopupTitle option, when we use the option it will be automatically 'convert' to dropDownOptions.showTitle. So test config is smth like { showTItle: false, showTitle: true }

Reply:
Does it mean that we have changed the priority of the `dropDownOptions.showTitle` option over `showPopupTitle`?

Reply:
As far as:
- DropDownEditor.dropDownOptions was not documented.
- A case when both options are specified is nonsense.

Let it be.